From 2aedb98b7c0b60ebe2b347b5a511861146b2062d Mon Sep 17 00:00:00 2001
From: Jakub Jirutka <jakub@jirutka.cz>
Date: Sat, 21 Jul 2018 12:27:28 +0200
Subject: [PATCH] fix ax_with_curses_extra to find <panel.h>

These autohells macros do not find panel.h in the root of the library
path when ncursesw variant is detected. This patch fixes the macros to
search both ncursesw/ subdirectory and the root of the library path.

Based on https://github.com/crosstool-ng/crosstool-ng/blob/master/m4/ax_with_curses_extra.m4

Upstream-Issue: https://github.com/okbob/pspg/pull/72
---
 tools/m4_ax_with_curses_extra.m4 | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/tools/m4_ax_with_curses_extra.m4 b/tools/m4_ax_with_curses_extra.m4
index 7056d68..81f5b79 100644
--- a/tools/m4_ax_with_curses_extra.m4
+++ b/tools/m4_ax_with_curses_extra.m4
@@ -212,6 +212,9 @@ AC_DEFUN([_AX_WITH_CURSES_EXTRA], [
     ], [
         AS_IF([test "x$ax_cv_curses_which" = xncursesw], [
             _AX_WITH_CURSES_CHECKEXTRA([$1], [$2], [$3], [ncursesw/$4], [$5])
+            AS_IF([test x$[]ax_cv_[]m4_tolower($1) != "xyes"], [
+                _AX_WITH_CURSES_CHECKEXTRA([$1], [$2], [$3], [$4], [$6])
+            ])
         ], [test "x$ax_cv_curses_which" = xncurses], [
             _AX_WITH_CURSES_CHECKEXTRA([$1], [$2], [$3], [$4], [$6])
             AS_IF([test x$[]ax_cv_[]m4_tolower($1) != "xyes"], [
